### PR TITLE
bpo-32114: Updated the documentation for get_event_loop to reflect the policy change

### PR DESCRIPTION
**Pull request by pablogsal**
https://bugs.python.org/issue32114

### CODE REVIEW

File: Doc/library/asyncio-eventloops.rst [hunk 1]
```diff
@@ -189,10 +189,14 @@ An event loop policy must implement the following interface:
...
 The default policy defines context as the current thread, and manages an event
-loop per thread that interacts with :mod:`asyncio`.  If the current thread
-doesn't already have an event loop associated with it, the default policy's
-:meth:`~AbstractEventLoopPolicy.get_event_loop` method creates one when
-called from the main thread, but raises :exc:`RuntimeError` otherwise.
+loop per thread that interacts with :mod:`asyncio`. An exception to this rule
+happens when :meth:`~AbstractEventLoopPolicy.get_event_loop` is called from a
+running future/coroutine, in which case it will return the current loop
+running that future/coroutine.
+If the current thread doesn't already have an event loop associated with it,
```
Comment (merwok) [inline Doc/library/asyncio-eventloops.rst]: Add an empty line before this to make a new paragraph.

Comment (pablogsal) [reply Doc/library/asyncio-eventloops.rst]: Done in d9a8a16

**Comment by pablogsal** [comment]
@1st1 Can you check that these changes to the documentation are correct?

**Review by 1st1 — APPROVED** [review]

**Comment by 1st1** [comment]
Merged. Thanks for the PR!

**Pull request merged**